server/app/services/UserService.ts

```diff
@@ -35,4 +35,9 @@ export class UserService {
 
     return UserEntity.buildUser(user);
   }
+
+  public async getUsers(): Promise<IUserEntity[]> {
+    const users: IUser[] = await this.userRepo.getList();
+    return await UserEntity.buildUsers(users);
```

Comment:
I don't think you need await since the buildUsers method doesn't return a Promise

Reply:
David is right!